src/sentry/tasks/store.py

```diff
@@ -771,6 +771,39 @@ def _do_save_event(
                     "events.time-to-process", time() - start_time, instance=data["platform"]
                 )
 
+            time_synthetic_monitoring_event(data, project_id, start_time)
+
+
+def time_synthetic_monitoring_event(data, project_id, start_time):
```

Comment:
Please let's add a doc string to explain what we do here.

Reply:
👍